src/main/java/frc/robot/commands/CmdManipulatorResetWrist.java

```diff
@@ -0,0 +1,42 @@
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.Constants;
+import frc.robot.subsystems.Manipulator;
+import frc.robot.subsystems.RobotLog;
+
+public class CmdManipulatorResetWrist extends CommandBase {
+    
+    private final Manipulator _manipulatorSubsystem;
+
+    public CmdManipulatorResetWrist() 
+    {
+        _manipulatorSubsystem = Manipulator.getInstance();
+    }
+
+    @Override
+    public void initialize() 
+    {
+        RobotLog.getInstance().log("Initializing Manipulator Wrist Reset");
+    }
+
+    @Override
+    public void execute() 
+    {
+        _manipulatorSubsystem.setWristSpeed(0.25);
```

Comment:
You can move this to the initialize function and remove the execute function entirely